### PR TITLE
isaaclevin.presencelight version 5.4.11

### PR DESCRIPTION
**Pull request by isaacrlevin**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58946)

### CODE REVIEW

File: manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- silent
+UpgradeBehavior: install
+PackageFamilyName: 37828IsaacLevin.197278F15330A_jvewcxq8vj8qt
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
+  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
+- Architecture: arm64
+  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
+  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
```
Comment (Trenly) [inline manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.installer.yaml:22]:
```suggestion
- Architecture: arm64
  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
- Architecture: x86
  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/58946/files#r860472936

File: manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Architecture: x86
+  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
+  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
+- Architecture: x86
+  InstallerUrl: https://github.com/isaacrlevin/presencelight/releases/download/Desktop-v5.4.11/PresenceLight.Package_5.4.11.0_x64_x86_ARM64.appxbundle
+  InstallerSha256: B091AC731E7FE1434B4169D3F1433A4A4DB5389449188F26A63C3E11325E10C0
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.installer.yaml:28]: Duplicate
```suggestion
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 5.4.11
```
Comment (ItzLevvie) [inline manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.4.11.0
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also have to update the version folder.

**Review by ItzLevvie — APPROVED** [review]

**Comment by isaacrlevin** [comment]
@ItzLevvie if I have a similar schema going forward (basically replacing version and hash) I should be able to automate this correct?

**Pull request merged**